base_classes/NXresolution.nxdl.xml

```diff
@@ -0,0 +1,125 @@
+    <doc>
+         Describes the resolution of a physical quantity.
+    </doc>
+    <field name="physical_quantity">
+        <doc>
+             The physical quantity of the resolution, e.g.,
+             energy, momentum, time, etc.
+        </doc>
+    </field>
+    <field name="type">
+        <doc>
+             The process by which the resolution was determined.
+        </doc>
+        <enumeration>
+            <item value="estimated"/>
+            <item value="derived"/>
+            <item value="calibrated"/>
+            <item value="other"/>
+        </enumeration>
+    </field>
+    <group name="note" type="NXnote">
+        <doc>
+             Additional details of the estimate or description of the calibration procedure
+        </doc>
+    </group>
+    <field name="resolution" type="NX_FLOAT" units="NX_ANY">
+        <doc>
+             The resolution of the physical quantity.
+        </doc>
+    </field>
+    <field name="resolution_errors" type="NX_FLOAT" units="NX_ANY">
+        <doc>
+             Standard deviation of the resolution of the physical quantity.
+        </doc>
+    </field>
+    <field name="relative_resolution" type="NX_FLOAT" units="NX_ANY">
+        <doc>
+             Ratio of the resolution at a specified measurand value to that measurand value,
+        </doc>
+    </field>
+    <field name="relative_resolution_errors" type="NX_FLOAT" units="NX_DIMENSIONLESS">
+        <doc>
+             Standard deviation of the relative resolution of the physical quantity.
+        </doc>
+    </field>
+    <group name="response_function" type="NXdata">
+        <doc>
+             The response of the instrument or part to a infinitesimally sharp input signal
+             along the physical quantity of this group.
+             This is also sometimes called instrument response function for time resolution or
+             point spread function for spatial response.
+             The resolution is typically determined by taking the full width at half maximum (FWHM)
+             of the response function.
+        </doc>
+        <field name="input" type="NX_FLOAT" units="NX_ANY">
+            <doc>
+                 The input axis or grid of the response function.
+                 The unit should match the one of the resolution field.
+            </doc>
+        </field>
+        <field name="magnitude" type="NX_FLOAT">
+            <doc>
+                 The magnitude of the response function corresponding to the points
+                 in the input axis or grid.
+                 This field should have the same dimensions as `input`.
+            </doc>
+        </field>
+    </group>
+    <field name="formula_SYMBOL" type="NX_CHAR">
+        <doc>
+             A symbol linking to another path in this appdef to be referred to from the
```

Comment:
This should be "A symbol linking to another path in the NeXus tree to be referred to from the..."